configs/nasfcos/nasfcos_r50_nashead_4x4_coco.py

```diff
@@ -0,0 +1,98 @@
+_base_ = [
+    '../_base_/datasets/coco_detection.py',
+    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
+]
+
+model = dict(
+    type='NASFCOS',
+    pretrained='open-mmlab://resnet50_caffe',
```

Comment:
In MMDetection V2.0, caffe_bgr as a pre-trained model is recommended. Do change the `img_norm_cfg` accordingly if you would like to use `resnet50_caffe_bgr`